### DISCUSSION TITLE
single sign-on and multi-logout

### DISCUSSION THREAD

**Discussion by MorganReid** [keycloak/keycloak#20382] [Q&A]
I created 2 clients on keyclaok, each client has its own login and logout. 
Now after integration, single sign-on and single sign-out can be realized through keycloak. 
Now I want to achieve single sign-on and multiple logouts, does keycloak support it? That is, after the user user visits clientA and then visits clientB, he will directly log in successfully. After the user logs out of clientA, he can continue to use it in clientB, that is, the user's session information will not be cleared by keycloak.

**Comment by darius-m** [comment]
I think you can achieve this by simply disabling single logout? Not sure if there's a better way, but you could do it by removing the logout URL from the client. Or configure the client to not send a logout to Keycloak.
